src/utils/fixLegacyQuery.ts

```diff
@@ -1,3 +1,7 @@
+import { ParsedQs } from 'qs'
```

Comment:
```suggestion
import type { ParsedQs } from 'qs'
```
was also thinking of adding that `import type` rule to linter

Reply:
nice